native/src/mc_interface/mc_interface_dump.h

```diff
@@ -16,18 +16,30 @@
 #define W_FLOAT(x) do { float d[] = { (float) x }; dump_out.write((char*) d, sizeof(float)); } while(false)
 #define W_DOUBLE(x) do { double d[] = { (double) x }; dump_out.write((char*) d, sizeof(double)); } while(false)
 
+#else
+
+#define DUMP_FUNC
+#define W_UINT32(x)
+#define W_FLOAT(x)
+#define W_DOUBLE(x)
```

Comment:
And do these even need to be defined if not used?

Reply:
Oh, are they only used in one file? I wasn't sure

Reply:
Fixed